examples/README.rst

```diff
@@ -8,6 +8,11 @@ examples soon. If you're interested in contributing to existing notebooks or
 adding new ones please consult the guide on
 :ref:`contributing_example_notebooks`.
 
+.. note::
+
+    The Fairlearn API is not completely stable, so if you want to run these
```

Comment:
Same comment as above. You should match versions, but not because this isn't stable, but because one always matches versions.